### PR TITLE
Added some basic assertion and actions

### PR DESCRIPTION
**Pull request by crschoen**

### CODE REVIEW

File: src/Behat/FlexibleMink/Context/FlexibleContext.php [hunk 1]
```diff
@@ -16,6 +19,8 @@ class FlexibleContext extends MinkContext
     use StoreContext;
     use JavaScriptContext;
 
+    const WARN_CANNOT_CHECK_VISIBLE = 'Could not determine element visibility. Driver does not support it.';
```
Comment (RedEnchilada) [inline src/Behat/FlexibleMink/Context/FlexibleContext.php]: This doesn't seem to be used anywhere in the context.

Comment (crschoen) [inline src/Behat/FlexibleMink/Context/FlexibleContext.php]: This line has been removed.

**Comment by crschoen** [comment]
@RedEnchilada, is this good to merge?

**Comment by RedEnchilada** [comment]
Good to go 👍

**Pull request merged**